postgres/init.sls

```diff
@@ -126,6 +147,7 @@ postgresql-db-{{ name }}:
     - lc_ctype: {{ db.get('lc_ctype', 'en_US.UTF8') }}
     - lc_collate: {{ db.get('lc_collate', 'en_US.UTF8') }}
     - template: {{ db.get('template', 'template0') }}
+    - tablespace: {{ db.get('tablespace', 'pg_default') }}
```

Comment:
Shouldn't this also require a:

{% if db.get('tablespace', 'pg_default') != 'pg_default' %}
   postgresql-tablespace-{{ name }}
{% endif %}

in the requirement section?
(or, alternately, a requires_in added to the tablespace state for the database?)

 +    - tablespace: {{ db.get('tablespace', 'pg_default') }}